frontend/src/components/form/tiptap/TiptapToolbarButton.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <v-btn icon tile height="40" width="42" color="black" class="rounded" v-on="$listeners">
```

Comment:
What happens with these fixed widths on mobile?

Reply:
I've optimized the toolbar, so that it is split into two rows. Buttons had also a fixed min width before.